src/extension.ts

```diff
@@ -72,10 +73,10 @@ export function activate(context: ExtensionContext) {
                 const started = await startLanguageServer(context, config);
                 languageServerIsRunning = started;
                 return languageServerIsRunning ? config.board.fqbn : undefined;
-            } catch (e) {
-                console.log(e);
+            } catch (err) {
+                console.error('Failed to start the language server.', err);
```

Comment:
The only effective change here is that I use `console.error` instead of `console.log` with an additional `Failed to start the language server.` message.